### PR TITLE
doc: npm install instructions for ubuntu

### PR DESCRIPTION
**Pull request by stephanegigandet**
Listing what I had to install to run rate my recipe locally on Ubuntu. (I know nothing about npm etc. so this might be obvious)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,20 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+
+Install NPM.
+
+On Ubuntu 22.04, the default installed version of NodeJS is too old.
+
+```bash
+sudo apt install npm
+sudo npm cache clean -f
+sudo npm install -g n
+sudo n stable
```
Comment (alexgarel) [inline README.md]: typo ?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,20 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+Clone the repository.
+
+Install NPM.
+
+On Ubuntu 22.04, the default installed version of NodeJS is too old.
+
```
Comment (alexfauquette) [inline README.md:10]: ### Setup JS env

If you're new to JavaScript environment, you might want to install [nvm](https://github.com/nvm-sh/nvm) which is a version manager for Node.js

You can follow the installation guide here: https://github.com/nvm-sh/nvm#installing-and-updating

After that, you will be able to run `nvm install <node version>` and `nvm use <node version>` to install and switch between node versions.

### Install the package manager

This project use `yarn` to manage library dependencies with 

```basj
npm install --global yarn
```

### Run the project

The following commend will install all the dependencies and run the project in `localhost:3000`
```bash
yarn dev
```

File: README.md [hunk 3]
```diff
@@ -2,7 +2,20 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+```bash
+sudo apt install npm
+sudo npm cache clean -f
+sudo npm install -g n
+sudo n stable
+```
+
+Run the development server:
```
Comment (alexfauquette) [inline README.md]: The current Readme is the one generated by project creation. I propose the following simplification:

```suggestion

### Setup JS env

If you're new to JavaScript environment, you might want to install [nvm](https://github.com/nvm-sh/nvm) which is a version manager for Node.js

You can follow the installation guide here: https://github.com/nvm-sh/nvm#installing-and-updating

After that, you will be able to run `nvm install <node version>` and `nvm use <node version>` to install and switch between node versions.

### Install the package manager

This project use `yarn` to manage library dependencies with 

```basj
npm install --global yarn
```

### Run the project

The following commend will install all the dependencies and run the project in `localhost:3000`
```bash
yarn dev
```
```

Comment (stephanegigandet) [reply README.md]: Thank you @alexfauquette

File: README.md [hunk 4]
```diff
@@ -2,27 +2,27 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
 
 ```bash
-sudo apt install npm
-sudo npm cache clean -f
-sudo npm install -g n
-sudo n stable
-```
+npm install --global yarn
+
+### Run the project
```
Comment (alexfauquette) [inline README.md]:
````suggestion
npm install --global yarn
```

### Run the project
````

**Review by alexfauquette — APPROVED** [review]

**Pull request merged**